aiographite/protocol.py

```diff
@@ -0,0 +1,50 @@
+from typing import Tuple, List
+import pickle
+import struct
+
+
+class PlaintextProtocol:
+
+    def format_data(self, metric: str, value: int, timestamp: int) -> str:
```

Comment:
We probably could set format_data as private method(_format_data). and use it only inside generate_message method.